File: Arrays/Easy/1480.RunningSumof1dArray/php/solution2.php
<?php
//solution 1

class Solution {

    /**
     * @param Integer[] $nums
     * @return Integer[]
     */
    function runningSum($nums) {
        $sum = [];
        for ($i = 0; $i < count($nums); $i++) {
            $sum[$i] = $sum[$i - 1] + $nums[$i];
        }

        return $sum;
    }
}

File: Practice/Easy/Array/268.MissingNumber/python/solution_2.py
# Gauses Formula
def miss_2(nums):
    n = len(nums)
    return n * (n + 1) // 2 - sum(nums)
    # sum([0, 1, 2, 3] - sum(nums) = 2


print(miss_2([3, 0, 1]))

